ssz-rs/src/bitlist.rs

```diff
@@ -88,7 +86,9 @@ impl<const N: usize> Bitlist<N> {
 
     fn pack_bits(&self) -> Result<Vec<u8>, MerkleizationError> {
         let mut data = vec![];
-        let _ = self.serialize_with_length(&mut data, false)?;
+        let _ = self
+            .serialize_with_length(&mut data, false)
+            .map_err(|_| MerkleizationError::SerializationError);
```

Comment:
this just ignores the error?

Reply:
Whoops, that isn't great! Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/b58448c30caa3cfc15bf696c020abb01001cac89.